FEXCore/Source/Interface/Core/OpcodeDispatcher/Crypto.cpp

```diff
@@ -388,7 +388,7 @@ void OpDispatchBuilder::VAESDecLastOp(OpcodeArgs) {
   Ref Key = LoadSource(FPRClass, Op, Op->Src[1], Op->Flags);
   Ref Result = _VAESDecLast(DstSize, State, Key, LoadZeroVector(DstSize));
 
-  StoreResult(FPRClass, Op, Result, -1);
+  StoreResult(FPRClass, Op, Result, OpSize::iInvalid);
```

Comment:
Perhaps we should have a three operand version of StoreResult, so that iInvalid doesn't need to be super prevalent all over the place

Reply:
I think that's reasonable, but I would want to leave that refactoring until after this change. Due to the large number of line changes I don't even want to change argument counts yet

Reply:
Makes sense